pkgs/applications/editors/jetbrains/darwin.nix

```diff
@@ -19,8 +19,9 @@ let
   loname = lib.toLower productShort;
 in
 stdenvNoCC.mkDerivation {
-  inherit pname meta src version plugins;
+  inherit pname src version plugins product;
```

Comment:
why are you adding product here?

Reply:
@FabianPonce can you answer my question and do a rebase so we can get this merged?

Reply:
It is referenced in `rootDir = if stdenv.hostPlatform.isDarwin then "Applications/${ide.product}.app/Contents" else meta.mainProgram;`. I put it in `passthru` for now but I'm not sure if it works.